jboss-kie-smartrouter/added/launch/jboss-kie-smartrouter.sh

```diff
@@ -47,6 +48,18 @@ function configure_mem_ratio() {
     export JAVA_INITIAL_MEM_RATIO=${JAVA_INITIAL_MEM_RATIO:-25}
 }
 
+function configure_logger_config_file() {
+  # use a default level if the env var is unset
+	local LOG_LEVEL=${LOG_LEVEL:-INFO}
```

Comment:
to not use same env names for local and exported.
and for local envs use lower case.